libraries/botbuilder-dialogs/src/dialogContext.ts

```diff
@@ -56,15 +56,22 @@ export interface DialogState {
 export class DialogContext {
     /**
       * Creates an new instance of the [DialogContext](xref:botbuilder-dialogs.DialogContext) class.
-      * 
-      * @remarks
-      * Passing in a dialog context instance will clone the dialog context.
       * @param dialogs The dialog set for which to create the dialog context.
       * @param contextOrDC The context object for the current turn of the bot.
       * @param state The state object to use to read and write dialog state to storage.
+      * @remarks
+      * Passing in a dialog context instance will clone the dialog context.
       */
     public constructor(dialogs: DialogSet, contextOrDC: TurnContext, state: DialogState);
```

Comment:
Missing comment for this constructor overload.
![image](https://user-images.githubusercontent.com/62260472/94688377-e95e7900-0303-11eb-8b10-c91dfcda123d.png)

Reply:
This error wasn't a part of the original linter report, should this be corrected regardless?

Reply:
Yea, I think we should add it.

Reply:
All right, description added. Thanks @sw-joelmut!